### ISSUE TITLE
CF configurator requests

### ISSUE THREAD

**Issue by Ozzy33** [cleanflight/cleanflight#496]
First post here, ClearFlight GUI suggestions

Add more items to the Adjustment Tab page;
Level PID,
ALT PID,
Mag P,
TPA,

Fix adjustments page to remember all settings, even when they are not enabled. (clarify:In the adjustments tab, I setup ~6 adjustments (CF 1.7.1 and GUI 0.61.0) , hit save (all fine), then only enable one of them hit save, all other adjustments are deleted (to see this, go to another tab then back to adjustments, all gone but the only enabled one stays).

Add to Receiver tab, indicators for when you move your sticks to show Forward/Backward, Left/Right, Idle/Full Throttle, like KK2 shows in Receiver page. Mainly getting the pitch correct is just a guess for new pilots.

In top box of GUI, add "date stamp" in front of time stamp (helpful when viewing screen shots in the forums, to tell how old it is).

In the firmware flasher, have check box (for each FC board) to pick your board, then it only shows FW for your board, then in the drop down, it will look cleaner.

Save buttons would be nice on top right of each page, no scrolling to get to them.

Button to save screen shot of settings from PID Tuning and Receiver. Easy to share PID's and Rx settings (RC rate, RC expo, thro expo), maybe wait on this one till the "Rates Overhaul #489" is done.

Even if none of these changes happen, I am way happier with CleanFlight over AttitudeFlight, Thanks  a million CF Team!

**Comment by andymac4182** [comment]
Hi @Ozzy33,

Can you please submit separate requests to the configurator issue list https://github.com/cleanflight/cleanflight-configurator/issues as each one will get worked on as a separate issue. 

Cheers
